examples/PySDM_examples/Kreidenweis_et_al_2003/simulation.py

```diff
@@ -25,12 +25,14 @@ def __init__(self, settings, products=None):
         builder.set_environment(env)
 
         attributes = env.init_attributes(
-            n_in_dv=settings.n_in_dv, kappa=settings.kappa, r_dry=settings.r_dry
+            n_in_dv=settings.n_in_dv,
+            kappa=settings.kappa,
+            r_dry=settings.r_dry,
+            include_dry_volume_in_attribute=False,
```

Comment:
@abulenok, @piotrbartman, what about such flag instead of the `del` statement in the outer scope?

Reply:
In general, I'm not a big fan of flags, but it's a much cleaner solution than the magical removal of attributes required for execution :)